### PR TITLE
svix-cli: init at 0.21.1

### PR DESCRIPTION
**Pull request by techknowlogick**
###### Description of changes

https://github.com/svix/svix-cli

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/by-name/sv/svix-cli/package.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    rev = revision;
+    hash = "sha256-bHcxhJs4Nu/hdiftQFZMx4M5iqFtpOzrsvXOgo9NlDc=";
+  };
+
+  vendorHash = "sha256-qSzEpxktdAV+mHa+586mKvpclCpXR6sE7HNcPZywd4s=";
+
+  # Increase minimum go version to 1.17 as the build fails with 1.16
+  # due to modules requiring code that was introduced in 1.17
+  # PR submitted upstream: https://github.com/svix/svix-cli/pull/103
+  patches = [ ./increase-minimum-go.patch ];
```
Comment (pbsds) [inline pkgs/by-name/sv/svix-cli/package.nix:20]: please `fetchpatch` the PR rather than vendor the patch

**Review by pbsds — COMMENTED** [review]
1 package built:
svix-cli


```
$ results/svix-cli/bin/svix-cli
A CLI to interact with the Svix API.

Usage:
  svix [command]

Available Commands:
  application     List, create & modify applications
  authentication  Manage authentication tasks such as getting dashboard URLs
  completion      Generate the autocompletion script for the specified shell
  endpoint        List, create & modify endpoints
  event-type      List, create & modify event types
  export          Export data from your Svix Organization
  help            Help about any command
  import          Import data to your Svix Organization
  integration     List, create & modify integrations
  listen          Forward webhook requests a local url
  login           Interactively configure your Svix API credentials
  message         List & create messages
  message-attempt List, lookup & resend message attempts
  open            Quickly open Svix pages in your browser
  verify          Verify the signature of a webhook message
  version         Get the version of the Svix CLI

Flags:
      --color enum   auto|always|never (default auto)
  -h, --help         help for svix
  -v, --version      Get the version of the Svix CLI

Use "svix [command] --help" for more information about a command.
```

**Comment by techknowlogick** [comment]
Thanks for the review @pbsds 🙏 This had slipped off my radar, but I've now made the changes from your feedback :)

**Comment by pbsds** [comment]
If a PR goes stale you're welcome to post to the thread on Discourse or Matrix and ask for reviews

**Pull request merged**